tests/flow/test_graph_info.py

```diff
@@ -83,16 +83,34 @@ def consumeStream(self, stream, drop=True):
 
         self.env.assertEquals(t, "stream")
 
-        # consume entire stream
-        events = self.conn.xrevrange(stream, '+', '-')
-
         # convert stream events to LoggedQueries
-        logged_queries = [LoggedQuery(e[1]) for e in events]
+        logged_queries = []
+        streams = {stream: '0-0'}
+
+        elapsed = 10
+        while len(logged_queries) < n_items and elapsed > 0:
+            # read messages from the stream
+            messages = self.conn.xread(streams, block=0)
```

Comment:
what's the meaning of `block=0` ?

Reply:
```c
block: number of milliseconds to wait, if nothing already present.
```

Reused this from prior function call (that I believe you wrote).